File: Games/Alex/src/de/hsh/alexander/actor/collectables/Invisible.java
package de.hsh.alexander.actor.collectables;

import common.actor.Actor;
import common.actor.Collectable;
import common.util.PlaySound;
import de.hsh.alexander.actor.ResourcePaths;

public class Invisible extends Collectable {

    public Invisible( double x, double y ) {
        super(x, y, ResourcePaths.Actor.Collectables.Invisible.invisiblePicture );
    }

    @Override
    public void wasCollected( Actor collector ) {
        PlaySound.playSound(ResourcePaths.Actor.Collectables.Invisible.invisibleSound );
        super.wasCollected( collector );
    }
}

File: Games/Alex/src/de/hsh/alexander/actor/collectables/Invisible.kt
package de.hsh.alexander.actor.collectables

import common.actor.Actor
import common.actor.Collectable
import common.util.PlaySound
import de.hsh.alexander.actor.ResourcePaths
import de.hsh.alexander.actor.ResourcePaths.Actor.Collectables.Invisible.invisiblePicture
import de.hsh.alexander.actor.ResourcePaths.Actor.Collectables.Invisible.invisibleSound

class Invisible(x: Double, y: Double) : Collectable(x, y, invisiblePicture) {
    override fun wasCollectedBy(collector: Actor) {
        PlaySound.playSound(invisibleSound)
        super.wasCollectedBy(collector)
    }
}
